homework3/src/main/java/first/FirstTask.java

```diff
@@ -0,0 +1,26 @@
+package first;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class FirstTask {
+
+    public static void main(String[] args) {
+
+        Random random = new Random();
+        Set<String> stringSet = new HashSet<>();
+        for (int i = 0; i < 5; i++) {
+            StringBuilder sb = new StringBuilder();
+            int len = random.nextInt(10) + 6;
```

Comment:
smart

Reply:
thanks, I thought it was very cool as well